test/forward_backward_compatibility/check_forward_backward_compatibility.py

```diff
@@ -276,6 +276,7 @@
     ("aten::sym_numel", datetime.date(2022, 10, 1)),
     # Distributed c10d ops are all going to be updated
     ("c10d::.*", datetime.date(2022, 10, 31)),
+    ("c10d::allgather_", datetime.date(2022, 10, 1)),
```

Comment:
nit: we don't need this as `c10d::.*` already included?

Reply:
`c10d::.*` is from from UCC developers. TBH, I don't know when they gonna remove those. If we act quickly and land this stack in a few days before UCC folks remove `c10d::*`, then yep, we don't need this for `allgather_`.